deployment/helm/templates/rbac.yaml

```diff
@@ -46,27 +48,25 @@ apiVersion: 'rbac.authorization.k8s.io/v1'
 kind: 'ClusterRoleBinding'
 metadata:
   name: 'akri-controller-binding'
-  namespace: {{ .Release.Namespace }}
```

Comment:
Since we are following helm guidelines, we should remove these as part of updating our documentation with our namespace guidelines.

Reply:
Nice to remember! Will be sure to remove them there.